### PR TITLE
First draft of a readme

### PR DESCRIPTION
**Pull request by mjg123**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,37 +1,17 @@
...
 
-You can use the [editor on GitHub](https://github.com/StreamOfJava/streamofjava.github.io/edit/master/README.md) to maintain and preview the content for your website in Markdown files.
+It's a landing page for those of us who stream Java on Twitch and YouTube.
 
-Whenever you commit to this repository, GitHub Pages will run [Jekyll](https://jekyllrb.com/) to rebuild the pages in your site, from the content in your Markdown files.
+### Who's that?
 
-### Markdown
+  - Ted Young (JitterTed): [Twitch](https://www.twitch.tv/jitterted) - [YouTube](https://www.youtube.com/channel/UCx8qMdZ6JoZgOcOQxWlyV8A) - [Twitter](https://twitter.com/jitterted)
+  - Nicolai Parlog (NipaFX): [Twitch](https://www.twitch.tv/nipafx) - [Twitter](https://twitter.com/nipafx)
```
Comment (nipafx) [inline README.md]:
```suggestion
  - Nicolai Parlog (nipafx): [Twitch](https://www.twitch.tv/nipafx) - [Twitter](https://twitter.com/nipafx)
```

File: README.md [hunk 2]
```diff
@@ -1,37 +1,17 @@
...
+  - Kevin Wittek (Kiview): [Twitch](https://www.twitch.tv/kiview) - [Twitter](https://twitter.com/kiview
```
Comment (nipafx) [inline README.md]:
```suggestion
  - Kevin Wittek (Kiview): [Twitch](https://www.twitch.tv/kiview) - [Twitter](https://twitter.com/kiview)
```

**Review by nipafx — COMMENTED** [review]
LGTM, two small edits:

**Review by kiview — APPROVED** [review]

**Comment by mjg123** [comment]
I'm happy for anyone to merge this, btw. I will do it if there's a 2nd approval - just want to give folks a chance to add themselves if they want to.

**Comment by tedyoung** [comment]
You can merge and I can submit a separate PR, or you can change my name to include the middle initial (`Ted M. Young`) and my shorter YouTube URL is https://www.youtube.com/jitterted/

Thanks!

**Pull request merged**